winml/test/scenario/cppwinrt/CustomOperatorProvider.h

```diff
@@ -26,6 +26,9 @@ struct CustomOperatorProvider :
 #elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_PC_APP)
         m_library = LoadPackagedLibrary(winml_dll_name.c_str(), 0 /*Reserved*/);
 #endif
+        if (!m_library) {
+          __fastfail(0);
```

Comment:
Should there be a more meaningful return code for __fastfail ?

Reply:
Used WINML_EXPECT_TRUE instead